wiki/BBCode/fr.md

```diff
@@ -16,18 +16,18 @@ Quand il n'y a pas de texte surligné, cliquer sur le bouton fait apparaitre des
 
 Les balises peuvent être combinées pour un formatage plus riche. L'ordre et l'imbrication des balises **doivent** être respecté quand vous combinez des balises. Le non respect de l'ordre / imbrication peut casser le formatage de la publication.
 
-Par exemple (faire attention à l'ordre du code) :
+Par exemple (faire attention à l'ordre du code):
```

Comment:
this is correct as-is, same with all of the other spaces before colons in French